src/app/pages/plan-cuentas/consulta-modificacion-apropiacion/list-modificacion-apropiacion.component.html

```diff
@@ -5,16 +5,13 @@
         <div *ngIf="!disabledVigencia">
 
           <div align="right">
-            {{
-              "GLOBAL.vigencia" | translate
-            }}
-            <nb-select nbTooltip="seleccione la vigencia" nbTooltipPlacement="top" id="select_vigencia" placeholder={{vigenciaSel}} [(selected)]="vigenciaSel" hero status="info">
+            {{ "GLOBAL.vigencia" | translate }}
+            <mat-progress-bar mode="indeterminate" *ngIf="vigenciaSel === '' || !vigencias || (vigencias && vigencias.length === 0)"></mat-progress-bar>
+            <nb-select *ngIf="vigenciaSel !== '' || vigencias.length > 0" nbTooltip="seleccione la vigencia" nbTooltipPlacement="top" id="select_vigencia" placeholder={{vigenciaSel}} [(selected)]="vigenciaSel" hero status="info">
               <!-- <nb-option value="vigenciaSel" (click)="onSelect('all')">Todas</nb-option> -->
```

Comment:
Eliminar esta línea comentada
```suggestion
```
... o descomentarla? (y luego traer el texto de i18n)
```suggestion
              <nb-option value="vigenciaSel" (click)="onSelect('all')">Todas</nb-option>
```

Reply:
Se elimina ya que la funcionalidad no está desarrollada